charts/trino/values.yaml

```diff
@@ -173,6 +173,31 @@ resourceGroups: {}
   #       ]
   #     }
 
+sessionPropertyConfig: {}
```

Comment:
@joshuasu-async sorry for the delay. I recently added a more complete test in https://github.com/trinodb/charts/blob/main/test-values.yaml. Can you update it to enable this config so we know this works correctly?

Also, please rebase and see the updated way of formatting examples here. We now include them in the README, so they should be formatted using Markdown.